todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/poc/PocActivity.java

```diff
@@ -0,0 +1,93 @@
+package com.example.android.architecture.blueprints.todoapp.poc;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.design.widget.NavigationView;
+import android.support.v4.app.NavUtils;
+import android.support.v4.view.GravityCompat;
+import android.support.v4.widget.DrawerLayout;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.MenuItem;
+
+import com.example.android.architecture.blueprints.todoapp.Injection;
+import com.example.android.architecture.blueprints.todoapp.R;
+import com.example.android.architecture.blueprints.todoapp.statistics.StatisticsActivity;
+import com.example.android.architecture.blueprints.todoapp.util.ActivityUtils;
+
+/**
+ * Created by user on 6/13/2017.
+ */
+
+public class PocActivity extends AppCompatActivity {
+
+    PocPresenter mPocPresenter;
+    DrawerLayout mDrawerLayout;
+    NavigationView mNavigationView;
+
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.pocs_act);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        ActionBar ab = getSupportActionBar();
+
+        // customize up icon to be drawable
+        ab.setDisplayHomeAsUpEnabled(true);
+        ab.setHomeAsUpIndicator(R.drawable.ic_menu);
+
+        mDrawerLayout = (DrawerLayout) findViewById(R.id.drawer_layout);
+        mNavigationView = (NavigationView) findViewById(R.id.nav_view);
+        if (mNavigationView != null)
+            setupDrawerContent(mDrawerLayout);
+
+        PocFragment pocFragment = (PocFragment) getSupportFragmentManager().findFragmentById(R.id.contentFrame);
+
+        if (pocFragment == null) {
+            pocFragment = PocFragment.newInstance();
+            ActivityUtils.addFragmentToActivity(getSupportFragmentManager(), pocFragment, R.id.contentFrame);
+        }
+        mPocPresenter = new PocPresenter(Injection.providePocRepo(), pocFragment);
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
```

Comment:
In the future, this should be implemented in our BaseActivity instead in order to avoid duplication.